sdk/servicebus/Azure.Messaging.ServiceBus/src/Primitives/ServiceBusModelFactory.cs

```diff
@@ -41,7 +41,8 @@ public static class ServiceBusModelFactory
             long sequenceNumber = -1,
             string deadLetterSource = default,
             long enqueuedSequenceNumber = default,
-            DateTimeOffset enqueuedTime = default)
+            DateTimeOffset enqueuedTime = default,
+            MessageState messageState = default)
```

Comment:
instead of adding the new param here, we will need to mark this as EditorBrowsableState.Never, make all params required, and add a new method.

Reply:
Thanks for the explanation. Will do